plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/IngestModule.java

```diff
@@ -52,41 +32,12 @@ public IngestModule(boolean ingestEnabled) {
 
     @Override
     protected void configure() {
-        // Even if ingest isn't enable we still need to make sure that rest requests with pipeline
+        // Even if ingest isn't enabled we still need to make sure that rest requests with pipeline
         // param copy the pipeline into the context, so that in IngestDisabledActionFilter
         // index/bulk requests can be failed
         binder().bind(IngestRestFilter.class).asEagerSingleton();
```

Comment:
Agreed and in the case of the `IngestRestFilter` this is easy. I briefly looked at it, but for other rest filter implementations it looked difficult to move away from guice, due to the fact that all kinds of services are wired to these implementations.